assets/generated_samples_list.json

```diff
@@ -992,7 +1021,7 @@
       "OGC",
       "WMS",
       "web map service",
-      "ArcGISMapViewController.identifyLayer",
+      "ArcGISMapViewController.identifyLayerAsync",
```

Comment:
Should not have `Async`

Reply:
That was a lingerer from a sample I did and missed. You could merge `v.next` and re-run the script as I've merged that now.

Reply:
script re-runned